### ISSUE TITLE
[feature] : suggest the design for career page

### ISSUE THREAD

**Issue by Sonichigo** [keploy/keploy#2069]
Keploy always has few job opening throughout the year, but is difficult to manage and track with forms and such. Therefore, it is necessary to have a keploy's career page which would contain - Job Title, Description, Resume & Cover Letter Submission, Tech Stack Preferred, Year of experience as fields.

Inspiration can be taken from other OSS companies and projects.

**Comment by charu1603** [comment]
@Swpn0neel I want to contribute to this issue. Can I work on this?

**Comment by Swpn0neel** [comment]
Please go ahead! @charu1603

**Comment by charu1603** [comment]
I’ve tried designing a career page and used Keploy's color theme for it.
Could you please take a look and provide your feedback?
@Sonichigo 
Figma link:(https://www.figma.com/design/SD8McEmOrhE5WuK4ZGit33/career-page?node-id=0-1&t=T1D2bt2knngOw9Ni-1)

**Comment by Swpn0neel** [comment]
Some immediate feedbacks:
1. The design of the application form should be improved
2. "We are hiring" should be displayed on hover, we can display the "Career" option by a blinking exclamation mark or something like that, to highlight it!

**Comment by charu1603** [comment]
@Swpn0neel @Sonichigo  please take a look and provide your feedback (https://www.figma.com/design/SD8McEmOrhE5WuK4ZGit33/career-page?node-id=0-1&t=oYy2154DMWJGz6n2-1 )

**Comment by Swpn0neel** [comment]
The inherent problem with this kind of Form design is that, once you fill in a field, you can't see what it was actually meant for. So kindly consider changing it!

![image](https://github.com/user-attachments/assets/30088397-1e55-4d90-8a2d-847f1d5a446e)

And also, we have to highlight somehow if we're currently hiring! Right now, you've removed it completely, other than modifying it according to the feedback!

**Comment by Swpn0neel** [comment]
Any update @charu1603 ?

**Comment by charu1603** [comment]
@Swpn0neel Thank you for your feedback regarding the form design and hiring notice.

1) The "We are hiring" notification has not been removed. It is displayed on hover over the "Careers" section in the navbar, . To view this, please tap on the prototype link provided. Blinking exclamation mark has been added . 
  
![Screenshot (620)](https://github.com/user-attachments/assets/4477c0d4-3c2d-4417-a3b2-f3db3c8fca1c)
(https://www.figma.com/proto/SD8McEmOrhE5WuK4ZGit33/career-page?node-id=9-191&t=1u9kCXNppwBEF1V1-1&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1&starting-point-node-id=9%3A191)

2) Improved the design of the application form.

**Comment by Sonichigo** [comment]
> @Swpn0neel Thank you for your feedback regarding the form design and hiring notice.
> 
> 1. The "We are hiring" notification has not been removed. It is displayed on hover over the "Careers" section in the navbar .
>    Blinking exclamation mark has been added .
>    Prototype link :-
> 
> ![Screenshot (620)](https://private-user-images.githubusercontent.com/96014425/351238655-4477c0d4-3c2d-4417-a3b2-f3db3c8fca1c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4ODQ5OTQsIm5iZiI6MTcyMTg4NDY5NCwicGF0aCI6Ii85NjAxNDQyNS8zNTEyMzg2NTUtNDQ3N2MwZDQtM2MyZC00NDE3LWEzYjItZjNkYjNjOGZjYTFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI1VDA1MTgxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM1ZTE1MjU1Njg3MTFjYTdiYzdkNDg3NDZjMmUyZTQ1ZjliNjE5MDY0YTg5NjlkZDg2YjZiMGZhYjFmNDVjMjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EET0BFjHZvFMbjnGM1OU4advTzkC2-ZKh3O87tX0r8U) (https://www.figma.com/proto/SD8McEmOrhE5WuK4ZGit33/career-page?node-id=9-191&t=1u9kCXNppwBEF1V1-1&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1&starting-point-node-id=9%3A191)
> 
> 2. Improved the design of the application form.
> 
> Could you please take a look and provide your feedback?

@charu1603 looks good! You can start working on the code part I'll increase the level and effort required for this.

**Comment by charu1603** [comment]
Thank you for the feedback @Sonichigo. I will start working on the code part. Could you please let me know should I clone the landing website of Keploy or work separately ?

**Comment by Sonichigo** [comment]
> Thank you for the feedback @Sonichigo. I will start working on the code part. Could you please let me know should I clone the landing website of Keploy or work separately ?

You can fork and clone and setup [keploy/website](https://github.com/keploy/website) repo locally, and start the code part :)

**Comment by Sonichigo** [comment]
@charu1603 any updates ?

**Comment by charu1603** [comment]
@Sonichigo, yes, I'm working on this issue. I'm working on the career page. Is it necessary to code both pages?

**Comment by Sonichigo** [comment]
> @Sonichigo, yes, I'm working on this issue. I'm working on the career page. Is it necessary to code both pages?

Just the first page in the figma file.

<img width="329" alt="image" src="https://github.com/user-attachments/assets/3b7152b5-8867-4e79-86c6-e2ce32027f4d">

**Comment by Sonichigo** [comment]
closing this since it's stale for long time
